ui/dashboards/src/components/TimeRangeControls/TimeRangeControls.tsx

```diff
@@ -32,6 +32,15 @@ export const TIME_OPTIONS: TimeOption[] = [
   { value: { pastDuration: '14d' }, display: 'Last 14 days' },
 ];
 
+export const REFRESH_TIME_OPTIONS: TimeOption[] = [
+  { value: { pastDuration: '0s' }, display: 'Never' },
```

Comment:
Let's go with 'Off' instead of 'Never' and '1m' instead of '60s'

Let me get some suggestions from @foxbat07 too on which longer intervals we'll need and I'll follow up!